src/blueapi/worker/task_worker.py

```diff
@@ -27,17 +32,29 @@
     WorkerEvent,
     WorkerState,
 )
-from .multithread import run_worker_in_own_thread
 from .task import Task
-from .worker import TrackableTask, Worker
 from .worker_errors import WorkerAlreadyStartedError, WorkerBusyError
 
 LOGGER = logging.getLogger(__name__)
 
 DEFAULT_START_STOP_TIMEOUT: float = 30.0
 
+T = TypeVar("T")
 
-class TaskWorker(Worker[Task]):
+
+class TrackableTask(BlueapiBaseModel, Generic[T]):
+    """
+    A representation of a task that the worker recognizes
+    """
+
+    task_id: str
+    task: T
+    is_complete: bool = False
+    is_pending: bool = True
+    errors: list[str] = Field(default_factory=list)
+
+
+class TaskWorker:
```

Comment:
Shall we just call it `Worker` now?

Reply:
worker is a generic python process term. https://docs.python.org/3/library/multiprocessing.html#using-a-pool-of-workers

taskworker is clearer